build.gradle.kts

```diff
@@ -54,13 +54,13 @@ java.sourceCompatibility = JavaVersion.VERSION_17
 plugins {
     id("io.gitlab.arturbosch.detekt") version "1.20.0"
     id("com.github.node-gradle.node") version "3.5.0" apply false
-    id("org.springframework.boot") version "2.7.5" apply false
+    id("org.springframework.boot") version "3.0.1" apply false
```

Comment:
Can you check if you can upgrade  ch.qos.logback:logback-classic along with spring boot?

Reply:
Done